skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ui/SkateConfigUI.kt

```diff
@@ -59,8 +59,10 @@ internal class SkateConfigUI(
           },
           setter = {
             if (File(it).isFile) {
-              settings.whatsNewFilePath =
-                LocalFileSystem.getInstance().findFileByPath(it)?.path.orEmpty()
+              val absolutePath = LocalFileSystem.getInstance().findFileByPath(it)?.path.orEmpty()
+              val projectPath = project.basePath ?: ""
+              val relativePath = absolutePath.removePrefix(projectPath).trimStart('/')
```

Comment:
```suggestion
              val relativePath = absolutePath.removePrefix(projectPath).removePrefix("/")
```

This is a more conventional API for this case